src/services/offer.js

```diff
@@ -48,8 +52,9 @@ class OfferService {
         return offer;
     }
 
-    async get() {
-        const offers = await Offer.find().current();
+    async get({ offset = 0, limit = 20 }) {
```

Comment:
Replace the 20 by accessing the static var instead

Reply:
Whoops, this must have been an oversight, really should be using it 🤦‍♂

Reply:
Fixed